specs/Transportation/CrowdFlowObserved/doc/spec.md

```diff
@@ -0,0 +1,120 @@
+# Crowd Flow Observed
+
+## Description
+
+An observation related to the movement of people at a certain place and time.
+
+## Data Model
+
++ `id`: Entity id. 
+    + It shall be `urn:ngsi-ld:CrowdFlowObserved:<identifier>` being `identifier` a unique identifier. 
+
++ `type`: Entity type. 
+    + It shall be equal to `CrowdFlowObserved`.
+    
++ `dateCreated` : Entity's creation timestamp. (`createdAt` in NGSI-LD)
```

Comment:
any reason why in NGSI-LD the attribute is different?

Reply:
standardization-wise ;)